requirements/base.txt

```diff
@@ -5,7 +5,7 @@ git+https://github.com/edx/XBlock.git@xblock-1.0.1#egg=XBlock==1.0.1
 git+https://github.com/edx/xblock-sdk.git@v0.1.4#egg=xblock-sdk==0.1.4
 
 # Third Party Requirements
-boto3>=1.4.4,<2.0.0
+boto>=2.48.0,<3.0.0
```

Comment:
As discussed with @jibsheet, we'll likely need a "rebuild from scratch" to get this working correctly.